todo_app/app.py

```diff
@@ -1,11 +1,44 @@
 from flask import Flask
+from flask import request
+from flask import redirect
+from flask import url_for
+from flask import render_template
```

Comment:
Down to personal preference, but note that we can import these on a single line like the local import if we prefer:
```python
from flask import Flask, request, redirect, url_for, render_template
```

Reply:
noted thanks - I can do a bit of clean up